Tiltfile

```diff
@@ -1,5 +1,7 @@
+load('.tanzu/tanzu_tilt_extensions.py', 'tanzu_k8s_yaml')
+
 custom_build('harbor-repo.vmware.com/tanzu_desktop/sample-app-java', 
-    "tanzu apps workload apply -f config/workload.yaml --local-path=. --yes && \
+    "tanzu apps workload apply -f config/workload.yaml --live-update --local-path=. --source-image=harbor-repo.vmware.com/tanzu_desktop/sample-app-java-souce --yes && \
```

Comment:
not sure if this was working before but there is a type in sample-app-java-source

Reply:
Good spot